cfme/common/provider_views.py

```diff
@@ -261,8 +261,12 @@ class ProviderTemplatesView(ProviderVmsTemplatesView):
     @property
     def is_displayed(self):
         title = VersionPicker({
-            Version.lowest(): '{name} (All VM Templates)'.format(name=self.context['object'].name),
-            '5.10': '{name} (All VM Templates and Images)'.format(name=self.context['object'].name),
+            Version.lowest(): '{name} (All VM Templates)'.format(
```

Comment:
Optional, consider cleaning up in follow up - the only difference here is part of the string, so you can separate out the duplicated logic of picking the object name or the object parent name.

```python
expected_name = getattr(self.context['object'], 'name', None) or getattr(context['object'].parent, 'name', None)
title = VersionPicker({Version.lowest(): f'{expected_name} (All VM Templates)', '5.10': f'{expected_name} (All VM Templates and Images)'})
```

Reply:
Sure, I agree. I was simply updating what was here with a small change without rewriting it. I will name a note to update this and also other instances of it that exist in the code (There are several).